docs/FAQs.md

```diff
@@ -0,0 +1,42 @@
+# GitHub for Jira FAQs
+
+### Permissions
+
+**Q:** The permission scope for code and metadata suggests data is read to synchronize development information. Is code stored on Jira? Should I be concerned that, were the app to be compromised, an actor exfiltrate all our code from GitHub?
```

Comment:
```suggestion
**Q:** The permission scope for code and metadata suggests data is read to synchronize development information. Is code stored on Jira? Should I be concerned that, were the app to be compromised, an actor can exfiltrate all our code from GitHub?
```

Reply:
Reading this again, 'could' would make more grammatical sense, would it not?